cdap-common/src/main/java/co/cask/cdap/common/twill/AbstractDistributedMasterServiceManager.java

```diff
@@ -179,4 +179,41 @@ protected final HttpResponseStatus checkGetStatus(String url) throws Exception {
       }
     }
   }
+
+  @Override
+  public boolean restartAllInstances() {
+    Preconditions.checkArgument(serviceName != null);
```

Comment:
Better to have a slightly more descriptive message here. E.g. "Name of service to restart is missing"